src/components/Inbox/index.jsx

```diff
@@ -1,11 +1,13 @@
-import EmailList from "../EmailList/Index";
+import EmailList from "../EmailList/Index.jsx"
```

Comment:
```suggestion
import EmailList from "../EmailList/"
```
We should have fixed the casing on these filenames so they should work now